vetro/patterns/hidden-404.php

```diff
@@ -6,18 +6,18 @@
  */
 ?>
 
-<!-- wp:group {"style":{"spacing":{"padding":{"top":"160px","right":"0px","bottom":"0px","left":"0px"},"blockGap":"0px","margin":{"top":"0px","bottom":"0px"}}},"layout":{"type":"default"}} -->
-<div class="wp-block-group" style="margin-top:0px;margin-bottom:0px;padding-top:160px;padding-right:0px;padding-bottom:0px;padding-left:0px"><!-- wp:group {"align":"wide","style":{"spacing":{"blockGap":"0px","padding":{"top":"0px","right":"0px","bottom":"0px","left":"0px"},"margin":{"top":"0px","bottom":"0px"}}}} -->
-<div class="wp-block-group alignwide" style="margin-top:0px;margin-bottom:0px;padding-top:0px;padding-right:0px;padding-bottom:0px;padding-left:0px"><!-- wp:heading {"textAlign":"left","align":"wide","style":{"typography":{"fontWeight":"600","lineHeight":"1","fontStyle":"normal","fontSize":"8rem"}},"className":"has-text-align-center"} -->
+<!-- wp:group {"tagName":"main","style":{"spacing":{"padding":{"top":"160px","bottom":"0px"},"blockGap":"0px","margin":{"top":"0px","bottom":"0px"}}},"layout":{"type":"constrained"}} -->
+<main class="wp-block-group" style="margin-top:0px;margin-bottom:0px;padding-top:160px;padding-bottom:0px"><!-- wp:group {"align":"wide","style":{"spacing":{"blockGap":"0px","padding":{"top":"0px","bottom":"0px"},"margin":{"top":"0px","bottom":"0px"}}}} -->
```

Comment:
This div has been changed to a main, but I think an ending main tag still needs to be added. I think it's probably the last /div on line 30 that needs to be changed.

Reply:
Oh good catch, thanks! Updated in [cc3d901](https://github.com/Automattic/themes/pull/6800/commits/cc3d9010fcf04da1383db4bff47fc62e8b43eba0)